docs/imviz/index.rst

```diff
@@ -21,10 +21,10 @@ and so on. (TODO: Add content.)
 Using Imviz
 -----------
 
-To run Imviz in a notebook::
+.. toctree::
```

Comment:
I cannot comment on the lines above but I think this paragraph on L17-L18 also needs some rephrasing... 😅 

`Imviz allows images to be easily displayed and examined. It supports WCS and so on. (TODO: Add content.)`

Reply:
Haha, good catch!